opencti-platform/opencti-graphql/src/config/conf.js

```diff
@@ -291,6 +300,26 @@ const auditLogger = winston.createLogger({
   transports: auditLogTransports,
 });
 
+// Setup support logs
+export const SUPPORT_LOG_RELATIVE_LOCAL_DIR = '.support';
+export const SUPPORT_LOG_FILE_PREFIX = 'support';
+const supportLogTransports = [];
```

Comment:
why not creating directly the supportLogTransports array ?

Reply:
Good question thanks, I change that.